sdk/tests/AccountIdTest.cc

```diff
@@ -59,13 +65,18 @@ TEST_F(AccountIdTest, ConstructWithShardRealmAccountNum)
   EXPECT_EQ(accountId.getShardNum(), getTestShardNum());
   EXPECT_EQ(accountId.getRealmNum(), getTestRealmNum());
   EXPECT_EQ(accountId.getAccountNum(), getTestAccountNum());
+
+  EXPECT_THROW(AccountId(getTestNumTooBig(), getTestRealmNum(), getTestAccountNum()), std::invalid_argument);
+  EXPECT_THROW(AccountId(getTestShardNum(), getTestNumTooBig(), getTestAccountNum()), std::invalid_argument);
+  EXPECT_THROW(AccountId(getTestShardNum(), getTestRealmNum(), getTestNumTooBig()), std::invalid_argument);
 }
 
 TEST_F(AccountIdTest, ConstructFromString)
```

Comment:
Currently, all of your successful fromString tests have only a single digit character. Since this deals with character parsing, I think it makes sense to also test with multicharacter numbers

Reply:
I changed the test realm and account numbers to be multi-character numbers.